source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -1,33 +1,38 @@
 3rd Party Libraries
 ===================
 
-Teams that are not using parts provided in the KoP will likely need to install external dependencies. Please view the below section to see information on adding an external library.
+Teams that are using non-PWM motor controllers or advanced sensors will most likely need to install external vendor dependencies. Please view the below section to see information on adding an external library.
 
-The Mechanism
--------------
+What Are Vendor Dependencies?
```

Comment:
I would restructure this a little. Libraries are how the vendors add their code. The vendor dependency is how the build system is told about the library.